src/solver/optimisation/opt_structure_probleme_a_resoudre.h

```diff
@@ -131,7 +128,8 @@ typedef struct {
 				   SPX_MATRICE_DE_BASE_SINGULIERE si on n'a pas pu construire de matrice de base reguliere,
 				                                  et dans ce cas il n'y a pas de solution
 			        */
-  PROBLEMES_SIMPLEXE ** ProblemesSpxDUneClasseDeManoeuvrabilite;
+
+  PROBLEMES_SIMPLEXE * ProblemesSpx;
```

Comment:
Do we still need multiple problems ?

Reply:
No, we don't. That's the reason why we use a PROBLEMES_SIMPLEXE * (1 star) instead of a PROBLEMES_SIMPLEXE ** (notice the 2 stars)